include/deal.II/base/std_cxx11/unique_ptr.h

```diff
@@ -18,6 +18,7 @@
 
 
 #include <deal.II/base/config.h>
+#include <boost/serialization/scoped_ptr.hpp>
```

Comment:
I think it would make more sense for this to be included below with the other `scoped_ptr` `#include`.